routes.js

```diff
@@ -135,6 +135,7 @@ const sharedRoutes = {
 
       return {
         title: isPR ? `Pull Request #${number}` : `Issue #${number}`,
+        headerLeft: navigation.state.params.headerLeft,
```

Comment:
Although this would essentially solve hiding the header on the left, have you tried swiping the screen from the left side? That might still allow the user to navigate back which we wouldn't want.

Also see comment 2 about possibly simplifying this flow 🕺